lib/bolt/util.rb

```diff
@@ -0,0 +1,54 @@
+module Bolt
+  module Util
+    # CODEREVIEW I hate mixing in random modules
+    class << self
+      def read_config_file(path, default_paths = nil, file_name = 'file')
+        logger = Logging.logger[self]
+        path_passed = path
+        if path.nil? && default_paths
+          found_default = default_paths.select { |p| File.exist?(p) }
+          if found_default.size > 1
+            logger.warn "Found #{file_name}s at #{found_default.join(', ')}, using the first"
+          end
+          # Use first found, fall back to first default and try to load even if it didn't exist
+          path = found_default.first || default_paths.first
+        end
+
+        path = File.expand_path(path)
+        # safe_load doesn't work with psych in ruby 2.0
+        # The user controls the configfile so this isn't a problem
+        # rubocop:disable YAMLLoad
+        File.open(path, "r:UTF-8") { |f| YAML.load(f.read) }
+      rescue Errno::ENOENT
+        if path_passed
+          raise Bolt::CLIError, "Could not read #{file_name} file: #{path}"
```

Comment:
Actually per my inventory comment, I think it makes more sense to just remove the word "file" from this strings. Otherwise there's another issue where config files will say "Could not read config file file".

Reply:
for now I think thats true. Long term I think inventory file may be meaningful